bin/machine-setup.sh

```diff
@@ -112,10 +112,17 @@ chmod +x update
 # Move the remainder of the files in the home directory over to the sol user
 sudo chown -R sol:sol ./*
 sudo mv ./* /home/sol
-# Move the systemd service file into /etc
+
+# Move the systemd service files into /etc
+sudo cp /home/sol/bin/sys-tuner.service /etc/systemd/system/sys-tuner.service
 sudo cp /home/sol/bin/"$NODE_TYPE".service /etc/systemd/system/sol.service
 sudo systemctl daemon-reload
 
+# Start the solana-sys-tuner service
+sudo systemctl start solana-sys-tuner
+sudo systemctl enable solana-sys-tuner
```

Comment:
You can combine both using `enable --now` :)

Reply:
fixed.